packages/commonwealth/client/scripts/views/pages/Communities/IdeaLaunchpad/QuickTokenLaunchForm/QuickTokenLaunchForm.tsx

```diff
@@ -450,22 +450,25 @@ export const QuickTokenLaunchForm = ({
                     placement="bottom"
                     content={`You can only generate a max of ${MAX_IDEAS_LIMIT} ideas.`}
                     renderTrigger={(handleInteraction) => (
-                      <CWButton
-                        iconLeft="brain"
-                        label="Randomize"
-                        containerClassName="ml-auto"
-                        type="button"
-                        disabled={
-                          isProcessingProfileImage ||
-                          isCreatingQuickToken ||
-                          isMaxTokenIdeaLimitReached
-                        }
-                        onClick={() => {
-                          generateIdea().catch(console.error);
-                        }}
+                      <div
```

Comment:
does it not work with button?

Reply:
yes in button mouseEnter events is not triggering that why tooltip show in mouseLeave and then hide in mouseLeave.

Reply:
it definitely should work in buttons. The one case that it does not is when the button is disabled, then the events are not getting through.

Reply:
yes button is disable after 5 try @masvelio